### PR TITLE
Added add/remove layers functions

### PR DESCRIPTION
**Pull request by AriGlockner**

### CODE REVIEW

File: stateFunctions.rkt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 (require "utils.rkt")
 (require "valueFunctions.rkt")
-
+(require rackunit)
```
Comment (1ethanhansen) [inline stateFunctions.rkt:7]: why are we requiring rackunit here?

File: tests.rkt [hunk 2]
```diff
@@ -39,3 +39,23 @@
...
+(check-equal? (remove-layer '(((z) (1)) ((x a) (10 2)))) '(((x a) (10 2))))
+
+; Add-Binding Tests
+(check-equal? (add-binding 'a 5 null) '(((a) (5))))
+(check-equal? (add-binding 'x 10 '((() ()))) '(((x) (10))))
+(check-equal? (add-binding 'a 2 '(((x) (10)))) '(((x a) (10 2))))
+(check-equal? (add-binding 'z 1 '((() ()) ((x a) (10 2)))) '(((z) (1)) ((x a) (10 2))))
+(check-equal? (add-binding 'b 'false '((() ()) ((z) (1)) ((x a) (10 2)))) '(((b) (false)) ((z) (1)) ((x a) (10 2))))
+
+; Remove-Binding Tests
```
Comment (1ethanhansen) [inline tests.rkt:61]:
```suggestion
```

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]
one minor change please

File: stateFunctions.rkt (hunk 1 above)
Comment (AriGlockner) [reply stateFunctions.rkt:7]: Oops. My bad. Let's fix that

**Pull request merged**